src/imageio/storage/picasa.c

```diff
@@ -945,8 +948,8 @@ static void ui_refresh_albums(dt_storage_picasa_gui_data_t *ui)
   }
   g_list_foreach(albumList, (GFunc)ui_refresh_albums_fill, model_album);
 
-  if(albumList != NULL) gtk_combo_box_set_active(ui->comboBox_album, 2);
-  // FIXME: get the albumid and set it in the PicasaCtx
+  if (albumList != NULL && current_index != -1)
+    gtk_combo_box_set_active(ui->comboBox_album, current_index);
```

Comment:
So what happens if during refresh the previously-selected album is now on the different position in the combobox?
I *think* that can happen?

Reply:
I think also that this can happen but I also assume this is the _very rare_ case.

If current index is _n_ and we still got at least _n_ elements in list another album will be selected (that's what happens now - we arbitraly select 2nd album). If we've got less than _n_ elements combobox will become empty.